docs/user-guide/how_to_use_mpi_plugin.md

```diff
@@ -0,0 +1,77 @@
+# MPI Plugin User Guide
+
+## Introduction
+
+**MPI plugin** is designed to optimize the user experience when running MPI jobs, it not only allows users to write less yaml, but also ensures the normal operation of MPI jobs.
+
+## How the MPI Plugin Works
+
+The MPI plugin will do three things:
+
+* Add the `dependsOn` field to the master task, so that the worker starts and then starts the master
+* Open ports used by MPI for all containers of the job
+* Force open `ssh` and `svc` plugins
```

Comment:
Do we also need to generate envs like `MPI_HOST` automatically?

Reply:
I've tried generating the `MPI_HOST` environment variable, but I have some difficulties.
The current way to generate `MPI_HOST` is **MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`**
This command will generate something like `MPI_HOST=host1,host2,host3`
But when I generate environment variables directly from configmap, (By configuring the pod's spec)
I get results like `MPI_HOST=host1 host2 host3`
It is missing comma. It cannot be used by mpi programs.

Do you have a better idea?

Reply:
what about generating MPI_HOST directly from job spec, just like generating the hostfile of the configmap ?

Reply:
It looks feasible and I will try to implement